src/editor.c

```diff
@@ -352,6 +353,20 @@ static gboolean on_editor_button_press_event(GtkWidget *widget, GdkEventButton *
 
 		return TRUE;
 	}
+
+	else if (event->button == 8 /*backwards*/ && editor_prefs.mouse_back_forward_enabled)
```

Comment:
Just a small thing: IMHO a comment in-between code is really not beautiful. Could you move that above or below the line? I guess in this case better below.